app/scripts/models/account.js

```diff
@@ -134,7 +134,7 @@ define(function (require, exports, module) {
           // Ignore UNAUTHORIZED errors; we'll just fetch again when needed
           // Otherwise report the error
           if (! AuthErrors.is(err, 'UNAUTHORIZED') && this._sentryMetrics) {
-            this._sentryMetrics.captureException(err);
+            this._sentryMetrics.captureException(new Error(_.isEmpty(err) ? 'Something went wrong!' : err));
```

Comment:
Should the `_.isEmpty` test be outside the `new Error` wrapper?

```js
_.isEmpty(err) ? new Error('Something went wrong!') : err
```

Reply:
@philbooth as confirmed in the past issue where you did `new Error(new Error(...))` it was okay to do that. 

If for some reason `err` is not an `Error` and I don't `new Error(err)` then I will not see the stack in Sentry and won't know that this is what is causing it.  
There might be a case where `isEmpty` is not the right solution, this way we know. 

I can reformat the code though to make it easier to read if that helps